file.go

```diff
@@ -129,7 +137,13 @@ func MakeGitHubZipFileRequest(gitHubCommit GitHubCommit, gitHubToken string) (*h
 		return request, fmt.Errorf("Neither a GitCommitSha nor a GitTag nor a BranchName were specified so impossible to identify a specific commit to download.")
 	}
 
-	url := fmt.Sprintf("https://api.github.com/repos/%s/%s/zipball/%s", gitHubCommit.Repo.Owner, gitHubCommit.Repo.Name, gitRef)
+	var url string
+	if instance.ApiUrl != "api.github.com" {
+		url = fmt.Sprintf("https://"+instance.ApiUrl+"/repos/%s/%s/zipball/%s", gitHubCommit.Repo.Owner, gitHubCommit.Repo.Name, gitRef)
+	} else {
+		url = fmt.Sprintf("https://api.github.com/repos/%s/%s/zipball/%s", gitHubCommit.Repo.Owner, gitHubCommit.Repo.Name, gitRef)
+	}
```

Comment:
Can this be simplified to:

```
url := fmt.Sprintf("https://%s/repos/%s/%s/zipball/%s", instance.ApiUrl, gitHubCommit.Repo.Owner, gitHubCommit.Repo.Name, gitRef)
```

Since in the `else` clause, we know `instance.ApiUrl = api.github.com`?

Reply:
Oh gosh, you're right. I was too deep into the idiom of the code that was already there, but yep, this is much cleaner! Fixed in 706117f853a8ae7fad577f4ad016a6550747c7ff.